www.foia.gov/_sass/_layout.scss

```diff
@@ -1,2 +1,6 @@
 @import 'layouts/sidenav';
 @import 'layouts/sidebar';
+
+#main {
+  min-height: 600px;
+}
```

Comment:
Curious why this needed to be added back in?

Reply:
lol, I keep adding it back in because when the input is the only thing on the page, the page is too short (just visually). We should remove it _for realz_ in the landing page story. Once you move the sidebar to the other non-landing page, the landing page will be too short again.